### PR TITLE
[YUNIKORN-1372] Get Involved communication channels page broken for community page

### PR DESCRIPTION
**Pull request by jagadeesanas2**
Community reporting_issues page link on the website is broken.

They point to and getting URL was not found on this server message. 
https://yunikorn.apache.org/community/reporting_issues/get_involved#communication-channels

Fixed by pointing to the right URL:
https://yunikorn.apache.org/community/get_involved#communication-channels

### CODE REVIEW

**Review by sunilgovind — APPROVED** [review]

**Review by 0yukali0 — APPROVED** [review]
LGTM

File: src/pages/community/reporting_issues.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ Please do not disclose the issue to any public forum before working with the sec
 If you have an issue with YuniKorn operation, please follow these guidelines:
 
 If you are having an issue with setup, configuration, or some other form of behavior not matching your expectation, join the user mailing list and ask your questions in that forum.
-See the [Get Involved](get_involved#communication-channels) page for information on mailing lists.
+See the [Get Involved](https://yunikorn.apache.org/community/get_involved#communication-channels) page for information on mailing lists.
```
Comment (wilfred-s) [inline src/pages/community/reporting_issues.md:37]: We should use a URL without the base: 
```
[Get Involved](/community/get_involved#communication-channels)
```
That is what we use everywhere else.

**Review by wilfred-s — CHANGES_REQUESTED** [review]
The URL

File: src/pages/community/reporting_issues.md (hunk 1 above)
Comment (jagadeesanas2) [reply src/pages/community/reporting_issues.md:37]: I will review all the URLs and update it

**Comment by jagadeesanas2** [comment]
Accidentally closed this PR. So created a new PR https://github.com/apache/yunikorn-site/pull/205

**Pull request closed without merging**